### DISCUSSION TITLE
Is there a way to config the delay time to get response from cache?

### DISCUSSION THREAD

**Discussion by phymo** [alovajs/alova#376] [Q&A]
Even when I get response from cache, there is delay time to make it looks like getting data from API server.
Is there a configuration to set the delay time to 0? to make it look like getting from local storage?

**Comment by MeetinaXD** [comment]
We did not add a delay to cached responses.

Can you describe how to reproduce it? Or provide a reproduction link.

**Reply by phymo** [reply]
> We did not add a delay to cached responses.
> 
> Can you describe how to reproduce it? Or provide a reproduction link.

Sorry, checked my code, the delay time is introduced from my code~
